contracts/src/v0.8/dev/automation/upkeeps/LinkAvailableBalanceMonitor.sol

```diff
@@ -6,26 +6,27 @@ import "../../../ConfirmedOwner.sol";
 import "../../../interfaces/automation/KeeperCompatibleInterface.sol";
 import "../../../vendor/openzeppelin-solidity/v4.7.0/contracts/security/Pausable.sol";
 import "../../../vendor/openzeppelin-solidity/v4.7.0/contracts/token/ERC20/IERC20.sol";
-import "../../../vendor/openzeppelin-solidity/v4.7.3/contracts/utils/structs/EnumerableSet.sol";
+import "../../../vendor/openzeppelin-solidity/v4.7.3/contracts/utils/structs/EnumerableMap.sol";
 
 interface IAggregatorProxy {
   function aggregator() external view returns (address);
 }
 
-interface IAggregator {
+interface ILinkAvailable {
   function linkAvailableForPayment() external view returns (int256 availableBalance);
-
-  function transmitters() external view returns (address[] memory);
```

Comment:
I was told this was important to avoid funding "dead" aggregators (for feeds at least) - has that requirement changed?